framework/math/math.cc

```diff
@@ -49,7 +49,7 @@ OmegaToPhiThetaSafe(const Vector3& omega)
 }
 
 void
-PrintMatrix(const MatDbl& A)
+PrintMatrix(const std::vector<std::vector<double>>& A)
```

Comment:
IMO this is an issue (but not related to this PR): `std::vector<std::vector<T>>` should not be used for a dense matrix type. Matrix is an object with all rows having the same number of columns. `std::vector<std::vector<T>>` allows by definition different number of columns in each row. Probably should open an issue if others agree.

Reply:
In general, I'm not a fan of `vector<vector....>>`. We need a proper matrix class and a proper multi-dimensional array. On a side note, this was the issue that `NDArray` was supposed to address, but it was never used the way it was intended.